README.md

```diff
@@ -49,6 +49,23 @@ $ colcon build --packages-up-to soss-fiware
     ```
     $ source install/local_setup.bash
     ```
+7. If you did not set up a running instance of Fiware's Orion Context Broker before, Docker provides with an easy way to configure it on your local machine, using a MongoDB database:
```

Comment:
I didn't tell you. We split long lines (120-140 characters) into several lines. Also, as a rule, we always split after a `.`